Update for prescriptive guidance

tmorrisonsfx opened the pull request:
Jira story: https://signalfuse.atlassian.net/browse/DOCS-1342

Live doc URL: https://docs.signalfx.com/en/latest/integrations/integrations-reference/integrations.conviva.html

@galpizarsfx PTAL.

galpizarsfx commented on conviva/README.md:
```diff
@@ -26,26 +26,28 @@ This monitor requires:
 ### INSTALLATION
 
-Install the latest version of the SignalFx Smart Agent as described [here](https://github.com/signalfx/integrations/tree/master/signalfx-agent).
+Install the latest version of the SignalFx Smart Agent as described in [Installation](https://github.com/signalfx/integrations/tree/master/signalfx-agent).
```
Since we have a Smart Agent tile, instead of linking to GitHub, let's link back to the Smart Agent tile. I can't seem to find an example of one tile linking to the agent tile, but it is possible. For instance, this is the markup for linking from the Google App Engine tile to the Google Cloud Platform tile. We may need to ask around or look at other tiles, but this is example: 

> Use SignalFx to monitor Google App Engine via [Google Cloud Platform](https://github.com/signalfx/integrations/tree/master/gcp)[](sfx_link:gcp).

galpizarsfx commented on conviva/README.md:
```diff
@@ -26,26 +26,28 @@ This monitor requires:
 
-Find and edit the SignalFx Smart Agent configuration file `agent.yaml` to configure the conviva monitor as described <a target="_blank" href="https://github.com/signalfx/signalfx-agent/blob/master/docs/monitors/conviva.md">here</a>. Use the metric parameters <a target="_blank" href="https://github.com/signalfx/integrations/blob/master/conviva/docs/conviva_metrics.md">here</a> as values for `metri ...
+1. Open the SignalFX Smart Agent `agent.yaml`configuration file.
+2. Edit the <a target="_blank" href="https://github.com/signalfx/signalfx-agent/blob/master/docs/monitors/conviva.md#overview">`monitors`</a> section.
```
If you look at the screenshot, since **monitors** has the  ` ` symbols, it makes the hyperlink not overly obvious. 
<img width="1166" alt="Screen Shot 2020-05-21 at 10 50 59 AM" src="https://user-images.githubusercontent.com/54906773/82578227-bb954680-9b51-11ea-8a02-0500af47d907.png">

You can either remove the ` ` so that the text is in blue, making it obvious that it's a link, or you can keep the  ` `, and then add a bullet point below the line that says, "To learn how to edit this monitor, see [Conviva](* To learn how to edit this monitor, see [conviva][conviva.md ).)

(Also, in the story, I provided a link to GitHub; that was my mistake, I noticed that there is a link to the doc site, so we should use that link instead.) 

First link: https://docs.signalfx.com/en/latest/integrations/agent/monitors/conviva.html

Second link: https://docs.signalfx.com/en/latest/integrations/agent/monitors/conviva.html#conviva-monitor-metric-parameters-and-metrics

galpizarsfx commented on conviva/README.md:
```diff
@@ -26,26 +26,28 @@ This monitor requires:
-Find and edit the SignalFx Smart Agent configuration file `agent.yaml` to configure the conviva monitor as described <a target="_blank" href="https://github.com/signalfx/signalfx-agent/blob/master/docs/monitors/conviva.md">here</a>. Use the metric parameters <a target="_blank" href="https://github.com/signalfx/integrations/blob/master/conviva/docs/conviva_metrics.md">here</a> as values for `metri ...
+1. Open the SignalFX Smart Agent `agent.yaml`configuration file.
+2. Edit the <a target="_blank" href="https://github.com/signalfx/signalfx-agent/blob/master/docs/monitors/conviva.md#overview">`monitors`</a> section.
+3. Add the <a target="_blank" href="https://github.com/signalfx/signalfx-agent/blob/master/docs/monitors/conviva.md#conviva-monitor-metric-parameters-and-metrics">metric parameters</a> as values for `metricParameter`.
```
Link to the doc site instead of to GitHub:

https://docs.signalfx.com/en/latest/integrations/agent/monitors/conviva.html#conviva-monitor-metric-parameters-and-metrics

tmorrisonsfx commented:
> Made some comments within the file :)

Thanks @galpizarsfx. I addressed your comments. The link for Smart Agent now points to the signalfx-agent repo.
